### DISCUSSION TITLE
Pair Extraordinaire, Galaxy Brain, Pull Shark

### DISCUSSION THREAD

**Discussion by als-creator** [community/community#30631] [Other Feature Feedback, Questions, & Ideas]
who can help me get this achievement? how to do it?

**Comment by melvincwng** [comment]
@als-creator Hey there, check out this link over here on how to get those achievements: https://github.com/Schweinepriester/github-profile-achievements

Hope this answers your question 😊

**Reply by als-creator** [reply]
this link is in my bookmarks, I asked who would help me get them. I added a couple commit but nothing came up

**Reply by victorlim4** [reply]
@als-creator - if you want to win the Pair Extraordinaire badge, follow the instructions in this doc:
[Creating a co-authored commit](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors)

then create a PR and just merge!

and to earn the Galaxy Brain Badge, you only need to answer 2 questions in discussion. and wait for the users to approve the answer.

To earn Pull Shark is much simpler, just create a PR in a repository (it can be your repo) and merge the changes.

now just go get them :)

**Reply by rajatsandeepsen** [reply]
galaxy brain can be achieved by replay to any two discussions

**Reply by zobaer53** [reply]
Tip for yolo badge.
1. Create a repo and add contributor from setting
2. Create PR and add that contributor the reviewer
3. Marge your PR

**Reply by als-creator** [reply]
> создайте PR в репозитории

I made and merged pull requests, I added a co-author to a commit in the web git and in the git desktop, for some reason the achievements do not appear, I read the instructions for obtaining achievements

**Reply by als-creator** [reply]
> Совет для значка yolo.
> 
> 1. Создайте репозиторий и добавьте участника из настройки
> 2. Создайте PR и добавьте этого участника в качестве рецензента
> 3. Маржируйте свой ПИАР

Do I need to wait for some confirmation from the collaborator after the paired commit? I don't get the icon after commit

**Comment by HalemoGPA** [comment]
To get galaxy brain, you have to answer two questions in any discussion except yours.
